src/index.js

```diff
@@ -98,26 +95,29 @@ function readAndPrintVersionFromPackageJson() {
  * 1. Environment Variable
  * 2. (key,value) from opts object
  * 3. Default False ( points to production URL )
- * @param {{}} [opts={}] - optional various options to override default EXHORT_DEV_MODE and DEV_EXHORT_BACKEND_URL.
+ * @param {{EXHORT_DEBUG?: string | undefined; EXHORT_BACKEND_URL?: string | undefined}} [opts={}]
```

Comment:
Should we rename the `EXHORT` env vars into `TRUSTDA`?

Reply:
seems reasonable to me